clients/client-shell/cmds/d2g/d2g.go

```diff
@@ -82,6 +82,20 @@ func convert(cmd *cobra.Command, args []string) (err error) {
 		return fmt.Errorf("failed to convert input to a docker worker payload definition: %v", err)
 	}
 
+	d2gConfig := map[string]interface{}{
+		"allowChainOfTrust":     true,
+		"allowDisableSeccomp":   true,
+		"allowHostSharedMemory": true,
+		"allowInteractive":      true,
+		"allowKVM":              true,
+		"allowLoopbackAudio":    true,
+		"allowLoopbackVideo":    true,
+		"allowPrivileged":       true,
+		"allowPtrace":           true,
+		"allowTaskclusterProxy": true,
+		"containerEngine":       engine,
+	}
```

Comment:
Are you referring to the `engine` variable? Because that one is. All others I just defaulted to `true` to show the full d2g-translated payload. I _could_ add in flags for all other toggles, but I'd argue that no one uses this command on the cli anyway

Reply:
yeah, fair enough